client/src/app.ts

```diff
@@ -306,8 +340,47 @@ class WebRTCApp {
         this.cameraChevronBtn.classList.remove("active");
       }
     });
+
+    // Text chat handlers
+    this.inputArea.addEventListener("submit", (ev) => {
+      ev.preventDefault();
+      this.handleTextSubmit();
+    });
   }
 
+  async handleTextSubmit() {
+    const text = this.msgInput.value;
+    const message = text.trim();
+
+    if (message.length === 0) {
+      return; // Ignore empty messages
+    }
+
+    try {
+      await this.rtviClient?.action({
+        service: "llm",
+        action: "append_to_messages",
+        arguments: [
+          {
+            name: "messages",
+            value: [
+              {
+                role: "user",
+                content: message,
+              },
+            ],
+          },
+        ],
+      });
+      this.chatLog(message.trim(), "User");
```

Comment:
This is the part where I'm unsure what we should invoke by default from the UI. Like whether it's `action` or `sendMessage`, or, per @mattieruth new proposal, `appendUserContext` or `sendClientMessage`. 

Maybe we should make this configurable ? Does it make sense ?

Reply:
This is a great point to consider for the next version. 
I grabbed this from sesame Can we keep this as-is for now and update with updated prebuilt and client library?